tsconfig.json

```diff
@@ -29,6 +29,9 @@
       "@datapunt/asc-ui/": [
         "./asc-ui/src"
       ],
+      "@datapunt/asc-icons/": [
```

Comment:
Do we have a plan what to do with asc-assets? Not sure if we should deprecate or not, since not many ppl use it..

Reply:
asc-icons is removed